app/controllers/categories_controller.rb

```diff
@@ -26,7 +26,7 @@ def edit
   def create
     @category = Category.new(category_params)
 
-    if @category.save
```

Comment:
using before_save would be a little more railsy: http://api.rubyonrails.org/classes/ActiveRecord/Callbacks.html